Swiftfin/Views/SettingsView/UserDashboardView/DeviceDetailsView/Components/Sections/CustomDeviceNameSection.swift

```diff
@@ -0,0 +1,35 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import JellyfinAPI
+import SwiftUI
+
+// TODO: Enable for CustomNames for Devices with SDK Changes
+
+extension DeviceDetailsView {
+    struct CustomDeviceNameSection: View {
+        @Binding
+        var customName: String
+
+        // MARK: - Body
+
+        var body: some View {
+            Section {
+                TextField(
+                    L10n.name,
+                    text: $customName
+                )
+            } header: {
+                Text(L10n.customDeviceName)
+                // TODO: Remove Footer after SDK Changes
```

Comment:
If this just doesn't work, then I don't want to have it surfaced in the app. We can just comment out the place where this is used and have a todo to uncomment when we update the SDK.

Reply:
The SDK lets us update the customName and the customName shows up on Web and other places where this is used. But, our current version of the SDK doesn't have customName included. So we can **set** it but we can't **see** it. 

That'd be commenting out the edit button and and the textField. Do you want me to proceed?